webview-ui/src/utils/slash-commands.ts

```diff
@@ -12,7 +14,8 @@ export interface SlashCommand {
 // Create a function to get all supported slash commands
 export function getSupportedSlashCommands(
 	customModes?: any[],
-	workflowToggles: Record<string, boolean> = {}, // kilocode_change
+	localWorkflowToggles: ClineRulesToggles = {}, // kilocode_change
+	globalWorkflowToggles: ClineRulesToggles = {}, // kilocode_change
```

Comment:
This file doesn't exist in Roo, so I question the value of kilocode_change here.